### ISSUE TITLE
no-passed-in-event-handlers should not warn on the built-in input component

### ISSUE THREAD

**Issue by boris-petrov** [ember-template-lint/ember-template-lint#1030]
```handlebars
{{input type="text" value=this.value keyPress=this.onKeyPress}}
```

This gives an `Event handler methods like ``keyPress`` should not be passed in as a component arguments  no-passed-in-event-handlers` warning while it shouldn't - there is no other way to pass an event handler to the built-in `input` component.

**Comment by lifeart** [comment]
@boris-petrov try angle-bracket style for `input`

**Comment by boris-petrov** [comment]
@lifeart:

```handlebars
<Input type="text" @keyPress={{this.onKeyPress}} />
```

This gives the same warning.

**Comment by lifeart** [comment]
hm, https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-passed-in-event-handlers.md

**Comment by Turbo87** [comment]
hmm, yeah, that does seem like a reasonable suggestion to me 👍

**Comment by rwjblue** [comment]
Agree
